docs/UserGuide.md

```diff
@@ -178,7 +179,8 @@ Expected output:
 
 ### 3.4 Marking as done: done
 Marks a module as done to show that it has been completed.
-This can be done based on a module code or module name.
+This can be done based on a module code or module name.\
+If you mark a module as done again, the grade will be updated.
```

Comment:
Do you think this part can be made clearer like this "If you mark a module as done again, the new grade that has been entered will update your module's grade" ?

Reply:
Okay, thanks for the suggestion!